src/components/fixed-header/fixed-header-styles.module.scss

```
@@ -97,6 +98,49 @@ $icon-size: 25px;
     overflow: visible;
   }
 }
+.optionsToggle {
+  position: absolute;
+  right: 25px;
+  transform: rotate(90deg);
+  top: 70px;
+  cursor: pointer;
+  @include animationFunction($propertyToAnimate: all);
+  &.openList{
+    transform: rotate(270deg);
+  }
+}
+.dropdownWrapper {
+  position: relative;
+}
+
+.titleOptionsList {
```

I think you can add some styles to the vertical scrollbar. They don't look exactly like those on the design so you can adjust the color and padding if you want:
```suggestion
.titleOptionsList {
  @extend %verticalScrollbarStyled;
```

---

🙌 thanks